mypy/types.py

```diff
@@ -2203,7 +2203,11 @@ def visit_instance(self, t: Instance) -> str:
         if t.erased:
             s += '*'
         if t.args:
-            s += '[{}]'.format(self.list_str(t.args))
+            if t.type.fullname == 'builtins.tuple':
+                assert len(t.args) == 1
+                s += '[{}, ...]'.format(self.list_str(t.args))
```

Comment:
This can also be written as `.format(t.args[0].accept(self))`, but this seems prettier.